packages/designsystem/package.json

```diff
@@ -1,23 +1,28 @@
 {
   "name": "@locokit/designsystem",
-  "version": "0.8.0",
+  "version": "1.0.0-alpha.0",
   "description": "LocoKit Design System, components",
-  "main": "src/index.ts",
+  "author": "Mathieu Dartigues",
+  "license": "MIT",
+  "main": "dist/index.cjs.js",
+  "module": "dist/index.es.js",
+  "types": "dist/src/index.d.ts",
   "scripts": {
     "lint": "eslint .",
-    "story:dev": "histoire dev",
+    "dev": "histoire dev",
```

Comment:
Why did you remove the 'story' before?

Reply:
This is for convenience, by running `turbo run dev`, all scripts `dev` are runned simultaneously.
So the "histoire"book. I think it's useful ? WDYT ?

Reply:
That's right, in this case you don't need to run `turbo run dev`. 
You could just individual npm scripts, just like before.